examples/demo/package.json

```diff
@@ -26,9 +26,9 @@
         "ra-input-rich-text": "^5.0.0",
         "ra-language-english": "^5.0.0",
         "ra-language-french": "^5.0.0",
-        "react": "^18.3.1",
+        "react": "^19.0.0",
```

Comment:
NPM complains about several incompatibilities with React 19 by packages used in the demo:

react@19.0.0 deduped invalid: 
- "^16.8.0 || ^17.0.0 || ^18.0.0" from examples/demo/node_modules/@apollo/client
- "^16.0.0 || ^17.0.0 || ^18.0.0" from examples/demo/node_modules/recharts
- "^16.0.0 || ^17.0.0 || ^18.0.0" from examples/demo/node_modules/react-resize-detector
- "^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0" from examples/demo/node_modules/react-smooth

Can you check it still works, especially with `recharts` and `apollo`?

Can you check if upgrading those libs allows to fix the issue?

(I ran `npm list react` to see those warnings)

Reply:
Everything works fine (recharts and apollo). I upgraded their deps for react-19.